src/common/imageio_module.c

```diff
@@ -155,6 +155,7 @@ static int dt_imageio_load_modules_format(dt_imageio_t *iio)
     gchar *libname = g_module_build_path(plugindir, (const gchar *)plugin_name);
     if(dt_imageio_load_module_format(module, libname, plugin_name))
     {
+      g_free(libname);
```

Comment:
Same in `dt_imageio_load_modules_storage`.

Reply:
Good catch, now fixed;